src/Dependencies/Collections/RoslynEnumerable.cs

```diff
@@ -17,5 +18,35 @@ public static SegmentedList<TSource> ToSegmentedList<TSource>(this IEnumerable<T
 
             return new SegmentedList<TSource>(source);
         }
+
+        public static bool TryGetCount<T>(this IEnumerable<T> source, out int count)
```

Comment:
why do we have two of these?

Reply:
I originally added it to the compiler code and found out the collections code couldn't reference that. I then added it to the collections code but left it in the compiler code because I thought it might be useful. Going to go ahead and remove.